src/Database/Orville/Internal/MigrateIndex.hs

```diff
@@ -24,6 +24,7 @@ createIndex :: MonadOrville conn m => conn -> IndexDefinition -> m ()
 createIndex conn (IndexDefinition {..}) = do
   let ddl = intercalate " " [ "CREATE"
                             , if indexUnique then "UNIQUE" else ""
+                            , if indexConcurrently then "CONCURRENTLY" else ""
```

Comment:
This needs to go after `INDEX`: https://www.postgresql.org/docs/11/sql-createindex.html